docs/applications/deploy-application.md

```diff
@@ -12,3 +12,12 @@ links:
     related:
     featured:
 ---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Applications` card
+1. Click `View` in the `Applications` card to see a list of existing `Applications`
+1. Click `DEPLOY` on the application you want to deploy
+1. Choose the branch and the pipeline to be deployed, then click `Next`
+1. Review deploy information then click `Confirm`
+    > There should be a green notification popup saying "Deployment triggered, please wait"
```

Comment:
My suggestion is to avoid mentioning (as much as possible) some details that could change very often and would lead the documentation to become deprecated:
```suggestion
   > A notification popup will be displayed, confirming that the deployment has been triggered
```